converters/toZigbee.js

```diff
@@ -691,11 +691,21 @@ const converters = {
     light_colortemp: {
         key: ['color_temp', 'color_temp_percent'],
         convertSet: async (entity, key, value, meta) => {
+            const preset = {'warm': 454, 'neutral': 370, 'cool': 250};
```

Comment:
Can we base the min/max on the actual color temp values provided by the bulb? I think the default should be according the default color temp range (150-500) `{'warm': 500, 'neutral': 325, 'cool': 150}`

Would be good to add an additional function in `light.js` which can do this.

Reply:
I don't think that's a good idea, ideally they are the same across different bulbs.

If I set all mine to neutral, I'd expect them to be the same. (370 is ikea and Innr default)
cool and warm values I picked from ikea too and they are in range of all bulbs I have collected colorTempMin/Max data from.

Reply:
I see, agree with that 

> However, maybe we can add 2 extra presets min and max ? (that would be bulb dependent?)

Good idea, but perhaps `warmest` and `coolest` is more self explanatory?

Reply:
OK, just did min/max, will rename them now... I like those names better.